autogpt_platform/backend/linter.py

```diff
@@ -21,7 +22,7 @@ def lint():
 
 
 def format():
-    run("ruff", "check", "--fix", ".")
-    run("isort", "--profile", "black", ".")
-    run("black", ".")
-    run("pyright", ".")
+    run("ruff", "check", "--fix", *target_dirs)
+    run("isort", "--profile", "black", *target_dirs)
+    run("black", *target_dirs)
+    run("pyright", *target_dirs)
```

Comment:
This clashes with the linting I set up for `autogpt_libs` in #8584, which only uses `ruff` (and `pyright` but that's not a linter/formatter)

Reply:
Moving `backend` from 3 formatters to 1 is a backlog item which I avoided in #8584 to not cause more conflicts than necessary

Reply:
I have configured `ruff` in `autogpt_libs` to cover both `black` and `isort`'s responsibilities, so adding them back doesn't really make sense. There are also very minor differences between the behavior of `ruff` and `black`, which is another reason why I don't want to add back `black`.

The simpler fix is to revert lines 26 and 27:
```suggestion
    run("ruff", "check", "--fix", *target_dirs)
    run("isort", "--profile", "black")
    run("black")
    run("pyright", *target_dirs)
```

Reply:
Okay I see the code changes are gone now, updated